src/Exporter.php

```diff
@@ -212,7 +213,11 @@ public function toArray(mixed $value): array
     public function countProperties(object $value): int
     {
         if ($this->canBeReflected($value)) {
-            $numberOfProperties = count((new ReflectionObject($value))->getProperties());
+            if ($value instanceof \stdClass) {
+                $numberOfProperties = count((new ReflectionObject($value))->getProperties());
+            } else {
```

Comment:
when using `ReflectionClass` with a `\stdClass`, the test `ExporterTest::testShortenedExport` fails

thus the problem remains if a stdClass is a lazy object.